contributors/devel/sig-release/cherry-picks.md

```diff
@@ -89,6 +90,9 @@ patch release branches.
     release you want to cherry pick to. Cherry picks should be applied to all
     [active](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#detailed-release-history-for-active-branches)
     release branches where the fix is applicable.
+  
+  - When hub intimates you for your github password, provide the created github token rather than your actual github password. 
```

Comment:
nitpick: could you wrap this at 80 characters and also link to the issue referenced in the PR?

Reply:
Sure! Will do so.

Reply:
@hasheddan Hope it's better now?